src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -175,7 +175,7 @@ public int Available
                 // This may throw ObjectDisposedException.
                 SocketError errorCode = SocketPal.GetAvailable(_handle, out argp);
 
-                if (NetEventSource.IsEnabled) NetEventSource.Info(this, $"Interop.Winsock.ioctlsocket returns errorCode:{errorCode}");
+                if (NetEventSource.IsEnabled) NetEventSource.Info(this, $"GetAvailable returns errorCode:{errorCode}");
```

Comment:
Perhaps using "Available_get" is better since it conveys it is a property getter. I'm worried that using 'GetAvailable' is confusing since people might look for uses of a method called that.

Reply:
I chose the name based on the "SocketPal.GetAvailable" call that this is reporting on, rather than it being part of the get_Available accessor.  In general that seemed to be the pattern here, reporting on the interop being performed rather than on the containing method.